gta/dff_importer.py

```diff
@@ -594,8 +606,11 @@ def import_frames():
             # Note: I have not considered if frames could have parents
             # that have not yet been defined. If I come across such
             # a model, the code will be modified to support that
-          
-            if  frame.parent != -1:
+
+            # print(frame)
+            # print(frame.parent)
+
+            if  frame.parent != -1 and frame.parent in self.objects:
```

Comment:
I got index out of range here with frame.parent of 2. No idea what's going on here though.